x-pack/legacy/plugins/ml/public/application/data_frame_analytics/pages/analytics_management/hooks/use_create_analytics_form/state.ts

```diff
@@ -38,27 +38,33 @@ export enum JOB_TYPES {
   CLASSIFICATION = 'classification',
 }
 
+export interface Option {
+  label: string;
+}
```

Comment:
Maybe EUI exposes a similar type we can reuse?

Reply:
`EuiComboBoxOptionProps` ?

Reply:
Sounds good! Updated in 5fb02c50cabc18a94c4952adee9d279bea7b54ac